unordered_associative_task_1/student/include/impl.h

```diff
@@ -32,4 +48,22 @@ using playlist = std::vector<Song>;
  *
  * @return true if first playlist is anagram of second
  */
-bool is_same_content(const playlist& first_playlist, const playlist& second_playlist);
+bool is_same_content(const playlist& first_playlist, const playlist& second_playlist)
+{
+    std::unordered_set<Song, MySongHash> playlist1{first_playlist.begin(), first_playlist.end()};
+    std::unordered_set<Song, MySongHash> playlist2{second_playlist.begin(), second_playlist.end()};
+    
+    if (first_playlist.size() != second_playlist.size())
+        return false;
+
+    if (playlist1.size() != playlist2.size())
+        return false;
+
+    for (auto& it : first_playlist)
+    {
+        if (playlist2.count(it) == 0)
```

Comment:
will it work for following case?
{"1", "1", "2"}
{"2", "2", "1"}

unique songs are "1" and "2", size is equal, but the amount of tracks differs